ruby-challenges.rb

```diff
@@ -1,34 +1,92 @@
 # ASSESSMENT 4: Ruby Coding Practical Questions
-# MINASWAN ✌️
-
+# MINASWAN 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
+# A method that takes in a number
+def even_or_odd num1
+    # if number is even (num1 % 2 = 0), return it is even
+    if num1 % 2 == 0
```

Comment:
Logic looks great!  Checkout Ruby's built in methods .even? and .odd? for a cool approach to this.